app/services/workflow.rb

```diff
@@ -10,6 +10,14 @@ def self.create_unless_exists(druid, workflow_name, version: 1)
     workflow_client.create_workflow_by_name(druid, workflow_name, version: version, lane_id: 'low')
   end
 
+  # Checks if current version has any incomplete wf steps and there is a versionWF
+  # @return [Boolean] true if object is open for versioning
+  def self.open_for_versioning?(druid, version)
```

Comment:
DSA and argo both do a similar check.  should this be centralized into workflow client?  (not in this ticket necessarily, i'd assume in a follow on if the answer is yes)